cluster/saltbase/salt/docker/init.sls

```diff
@@ -99,6 +99,24 @@ net.ipv4.ip_forward:
 
 {% set storage_base='https://storage.googleapis.com/kubernetes-release/docker/' %}
 
+# Install docker on debian wheezy 7 VM
+{% if grains.cloud is defined and grains.cloud == 'gce'
+   and grains.os_family == 'Debian'
+   and grains.oscodename == 'wheezy'
+   and not salt['file.file_exists']('/usr/bin/docker') -%}
+  docker-engine:
+    pkg.installed:
+     - sources:
+         - docker-engine_1.9.0: http://apt.dockerproject.org/repo/pool/main/d/docker-engine/docker-engine_1.9.0-0~wheezy_amd64.deb
```

Comment:
About 5 rebases back, debian 7 wheezy backports was not working as the image itself was changed and did not have support for cgroup memory in the default grub. I see that updated now.

Reset this file to master.  You will not see this in the PR now.